service/collector.go

```diff
@@ -166,9 +166,13 @@ func (col *Collector) Shutdown() {
 	close(col.stopTestChan)
 }
 
-func (col *Collector) setupTelemetry(ballastSizeBytes uint64) error {
+func (col *Collector) setupTelemetry() error {
 	col.logger.Info("Setting up own telemetry...")
 
+	// Set memory ballast
+	ballast, ballastSizeBytes := col.createMemoryBallast()
+	runtime.KeepAlive(ballast)
```

Comment:
I think we'll only allow to setup memory ballast from ballast extension. I'll remove the whole ballast setup code from setupTelemetry later.

Reply:
I will update this PR to remove the ballast, then.